src/openrct2/ride/TrackDesignSave.cpp

```diff
@@ -32,9 +32,8 @@
 
 #include <algorithm>
 
-#define TRACK_MAX_SAVED_TILE_ELEMENTS 1500
-#define TRACK_NEARBY_SCENERY_DISTANCE 1
-#define TRACK_TD6_MAX_ELEMENTS 8192
```

Comment:
@duncanspumpkin this is kind of repeated here:
https://github.com/OpenRCT2/OpenRCT2/blob/e9bcad193ac098ecd8bb73db6fbb24fb94dd86b3/src/openrct2/ride/TrackDesign.cpp#L249-L251

I know it's not on the exporter code, but maybe it's what you meant?
`8192` is not found on any exporter code. If you know where to place it, I can change it as part of the PR

Reply:
Ah cool it's in a better location than i thought. Delete away then.

Reply:
Alright!